pkg/cmd/roachtest/test_runner.go

```diff
@@ -1399,6 +1399,9 @@ func (r *testRunner) collectArtifacts(
 		if err := c.CopyRoachprodState(ctx); err != nil {
 			t.L().Printf("failed to copy roachprod state: %s", err)
 		}
+		if err := c.FetchPebbleCheckpoints(ctx, t.L()); err != nil {
+			t.L().Printf("failed to fetch Pebble checkpoints: %s", err)
+		}
```

Comment:
Have you tested that this works?

Could probably use the `inconsistency` roachtest for that.

Reply:
Only manually. I'll add tests once we agree on the direction.